src/algorithms/ea.jl

```diff
@@ -15,40 +15,37 @@ Returns a [`Result`](@ref).
 """
 function oneplusone(f::Function, ind::AbstractVector, k_max::Integer, M::MutationMethod)
     fx = Inf  # works only on minimisation problems
-    for _ in 1:k_max
+    runtime = @elapsed @inbounds for _ in 1:k_max
         c = mutate(M, ind)
-        fx = f(ind)
-        fc = f(c)
-        if fc <= fx  # O(2 * k_max)  # minimisation problem
+        fx, fc = f(c), f(ind)
```

Comment:
This should be the other way around, right? `fc` is $f(c)$; the function evaluation of the child.

Reply:
Ok I see below that indeed this was just a mistake :) 
```suggestion
        fx, fc = f(ind), f(c)
```